sdk/monitor/monitor-opentelemetry-exporter/src/export/statsbeat/types.ts

```diff
@@ -101,7 +105,30 @@ export interface CommonStatsbeatProperties {
   attach: string;
 }
 
+export interface AttachStatsbeatProperties {
```

Comment:
We need to use "I" always before the name of all interfaces or not use it ever, looks like we have a mix here

Reply:
Chose to unify on not using the "I" here.